### DISCUSSION TITLE
Ant design package version update

### DISCUSSION THREAD

**Discussion by v4voloshyn** [refinedev/refine#5065] [Q&A] [answered]
Hello, everyone! Can anyone give me an answer to the following question: Can I update the `antd` package from version 5.0.5 to version 5.8.x (the latest) without further possible compatibility issues with `refinedev/core` or `refinedev/antd`? My current versions of the packages are: `"@refinedev/antd": "^5.34.0" `and `"@refinedev/antd": "^5.34.0"`.

**Comment by BatuhanW** [comment, marked as answer]
Hello @v4voloshyn sorry for the late reply!

AFAIK, we don't have any blocker to do that. You can just do it. And let us know if you come into any issues, we can help!
